fs/mmap/fs_rammap.c

```diff
@@ -81,15 +149,14 @@ struct fs_allmaps_s g_rammaps =
  *
  ****************************************************************************/
 
-int rammap(FAR struct file *filep, size_t length,
-           off_t offset, bool kernel, FAR void **mapped)
+int rammap(FAR struct file *filep, FAR struct mm_map_entry_s *map_entry,
```

Comment:
let's unify the term, either entry or map_entry, not both in the code base.

Reply:
done, it is just "entry" in all places